apps/web/src/pages/templates/editor/PreviewMobile.tsx

```diff
@@ -0,0 +1,138 @@
+import { createStyles, Group } from '@mantine/core';
+import { format } from 'date-fns';
+import Frame from 'react-frame-component';
+import { colors } from '../../../design-system';
+import { PreviewDateIcon } from './PreviewDateIcon';
+import { PreviewUserIcon } from './PreviewUserIcon';
+
+const useStyles = createStyles(() => ({
+  phone: {
+    display: 'block',
+    margin: 'auto',
+    width: '390px',
+    height: '740px',
+    position: 'relative',
+    borderColor: colors.B20,
+    borderWidth: '10px',
+    borderStyle: 'solid',
+    borderRadius: '40px',
+  },
+  cameraDent: {
+    background: colors.B20,
+    width: '148px',
+    height: '30px',
+    marginLeft: 'auto',
+    marginRight: 'auto',
+    borderRadius: '0 0 20px 20px',
+  },
+}));
+
+export const PreviewMobile = ({
+  integration,
+  subject,
+  content,
+}: {
+  integration: any;
+  subject: string;
+  content?: string;
+}) => {
+  const { classes } = useStyles();
+
+  return (
+    <>
+      <div className={classes.phone}>
+        <div className={classes.cameraDent}></div>
+        <div
+          style={{
+            width: '100%',
+            marginTop: '20px',
+            paddingLeft: '15px',
+            paddingRight: '15px',
+          }}
+        >
+          <Group
+            sx={{
+              height: '40px',
+            }}
+            spacing={20}
+            position="apart"
+          >
+            <div>
+              <Group
+                sx={{
+                  height: '40px',
+                }}
+                spacing={15}
+              >
+                <PreviewUserIcon />
+                <div>
+                  <div
+                    style={{
+                      marginBottom: '3px',
+                      fontWeight: 'bolder',
+                    }}
+                  >
+                    {subject}
+                  </div>
+                  <div
+                    style={{
+                      color: colors.B60,
+                      fontWeight: 'normal',
+                    }}
+                  >
+                    {integration?.credentials?.from || 'No active email integration'}
+                  </div>
+                </div>
+              </Group>
+            </div>
+            <div>
+              <div
+                style={{
+                  height: '20px',
+                  marginTop: '20px',
+                  color: colors.B60,
+                  fontWeight: 'normal',
```

Comment:
Again, maybe move it to `useStyles`. Not sure what name to suggest as I don't know what totally renders this.